lib/scala/docs-generator/src/main/scala/org/enso/docs/generator/Main.scala

```diff
@@ -157,8 +157,10 @@ object Main {
     val allDocJSFiles = allFiles.map { x =>
       val name = x.getPath
         .replace(".enso", ".js")
-        .replace("std-lib/", "std-lib/" + outDir + "/")
+        .replace("lib/Standard/", outDir + "/")
         .replace("Main.js", "index.js")
+        // TODO [RW] update this once library versions are changing
+        .replace("/0.1.0/", "/")
```

Comment:
@BinarySoftware can you have a look at it?

The docs seem to run, but I'm not exactly sure what these artifacts should look like so it is hard to tell if I didn't break something in a more subtle way. Would really appreciate if you could double check if these changes did not break the docs.

Reply:
why not just repace by regex?

from   `/0.1.0/` to `\/\d+\.\d+\.\d+\/`?

Reply:
Because currently the version is hardcoded anyway and once it is changed, this regex will not suffice (as the version can contain a `SNAPSHOT` suffix and other stuff). 

Since this will need to be changed anyway (or maybe will be completely changed before the version stops being hardcoded), I don't think a regex is necessary now.